tasks/ast_tools/src/generators/typescript.rs

```diff
@@ -27,8 +27,8 @@ impl Generator for TypescriptGenerator {
         let file = file!().replace('\\', "/");
         let mut content = format!(
             "\
-        		// To edit this generated file you have to edit `{file}`\n\
-        		// Auto-generated code, DO NOT EDIT DIRECTLY!\n\n\
+        		// Auto-generated code, DO NOT EDIT DIRECTLY!\n\
+        		// To edit this generated file you have to edit `{file}`\n\n\
```

Comment:
Indentation still looks weird here. Are there still tabs? If your IDE isn't letting you use spaces for some reason, no worries, I can fix in a follow-up.